kubetest/kops.go

```diff
@@ -137,6 +137,9 @@ func (k kops) Up() error {
 	if k.image != "" {
 		createArgs = append(createArgs, "--image", k.image)
 	}
+	if os.Getenv("KOPS_ARGS") != "" {
```

Comment:
Ok, I'm now working this now but having a lot of trouble plumbing --kops-args through the python scenario script->e2e-runner.sh->kubetest. Can you see anything obvious I'm doing wrong in the new PR code, any help would be appreciated!

printenv shows that E2E_OPT looks fine coming out of the python script and into e2e-runner.sh, kops-args value is quoted because there are spaces:
```
E2E_OPT=--deployment kops ... --kops-args="--dns private" ...
```
but later after various e2e_go_args are added, kubetest is ultimately called like this:
```
+(/workspace/e2e-runner.sh:179): main(): kubetest --deployment kops... '--kops-args="--dns' 'private"' ...
```

So I think there is some bash stuff happening that I don't quite understand how to get around.